doc/Installation/Installation-CentOS-7-Apache.md

```diff
@@ -0,0 +1,132 @@
+n> NOTE: These instructions assume you are the root user.  If you are not, prepend `sudo` to the shell commands (the ones that aren't at `mysql>` prompts) or temporarily become a user with root privileges with `sudo -s` or `sudo -i`.
+
+### DB Server ###
+
+> NOTE: Whilst we are working on ensuring LibreNMS is compatible with MySQL strict mode, for now, please disable this after mysql is installed.
+
+#### Install / Configure MySQL
+```bash
+yum install mariadb-server mariadb
+mysql -uroot -p
+```
+
+```sql
+CREATE DATABASE librenms;
+GRANT ALL PRIVILEGES ON librenms.*
+  TO 'librenms'@'localhost'
+  IDENTIFIED BY '<password>'
+;
+FLUSH PRIVILEGES;
+exit
+```
+
+`vim /etc/my.cnf`
+
+Within the [mysqld] section please add:
+
+```bash
+innodb_file_per_table=1
+sql-mode=""
+```
+
+```service mysql restart```
+
+### Web Server ###
+
+#### Install / Configure Apache
+
+`yum install epel-release`
+
+`yum install php70w php70w-cli php70w-gd php70w-mysql php70w-snmp php70w-pear php70w-curl php70w-common php70w-fpm nginx net-snmp mariadb ImageMagick jwhois nmap mtr rrdtool MySQL-python net-snmp-utils cronie php70w-mcrypt fping git`
+
+In `/etc/php.ini` ensure date.timezone is set to your preferred time zone.  See http://php.net/manual/en/timezones.php for a list of supported timezones.  Valid examples are: "America/New York", "Australia/Brisbane", "Etc/UTC".
+
+#### Add librenms user
+
+```bash
+useradd librenms -d /opt/librenms -M -r
+usermod -a -G librenms apache
+```
+
+#### Clone repo
+
+```bash
+cd /opt
+git clone https://github.com/librenms/librenms.git librenms
+```
+
+#### Web interface
+
+```bash
+cd /opt/librenms
+mkdir rrd logs
+chmod 775 rrd
+vim /etc/httpd/conf.d/librenms.conf
+```
+
+Add the following config:
+
+```apache
+<VirtualHost *:80>
+  DocumentRoot /opt/librenms/html/
+  ServerName  librenms.example.com
+  CustomLog /opt/librenms/logs/access_log combined
+  ErrorLog /opt/librenms/logs/error_log
+  AllowEncodedSlashes NoDecode
+  <Directory "/opt/librenms/html/">
+    Require all granted
+    AllowOverride All
+    Options FollowSymLinks MultiViews
+  </Directory>
+</VirtualHost>
+```
+
+```bash
+service httpd restart
+```
+
+> NOTE: If this is the only site you are hosting on this server (it should be :)) then you will need to disable the default site.
+
+`rm -f /etc/httpd/conf.d/welcome.conf`
+
+#### Web installer
+
+Now head to: http://librenms.example.com/install.php and follow the on-screen instructions.
```

Comment:
Permission issue when trying to let apache copy the config.php

"We couldn't create the config.php file.."